### PR TITLE
Quadrat: Fix post meta offset

### LINKED ISSUE #4748: Quadrat: Slight offset in tagline and post meta
I was checking Quadrat on mobile and there's a slight offset in tagline and post meta. 

The spacing is also too tight between post meta and the post title.

Captura de ecrã 2021-09-30, às 12 34 59

The misalignment is also happening in larger screens:

Captura de ecrã 2021-09-30, às 13 02 54

### PR DESCRIPTION
**Pull request by danieldudzic**
Closes #4748

### CODE REVIEW

**Comment by danieldudzic** [comment]
@jffng Do you think this approach is correct, here?

File: quadrat/sass/templates/_meta.scss [hunk 1]
```diff
@@ -28,6 +28,9 @@
 			background: transparent;
 			-webkit-mask-image: none;
 			mask-image: none;
+			height: 0;
```
Comment (jffng) [inline quadrat/sass/templates/_meta.scss]: Could we just use `content: none;` here and remove the rest of CSS specific to the `:before` element?

File: quadrat/block-template-parts/header.html [hunk 2]
```diff
@@ -2,7 +2,7 @@
 <header class="wp-block-group site-header" style="padding-bottom:170px">
 	<!-- wp:site-logo /-->
 	<!-- wp:site-title /-->
-	<!-- wp:site-tagline {"style":{"spacing":{"padding":{"top":"0px","right":"0px","bottom":"0px","left":"20px"},"margin":{"right":"auto"}},"typography":{"fontSize":"var(--wp--custom--font-sizes--tiny)"}}} /-->
+	<!-- wp:site-tagline {"style":{"spacing":{"padding":{"top":"0px","right":"0px","bottom":"0px","left":"0px"},"margin":{"right":"auto"}},"typography":{"fontSize":"var(--wp--custom--font-sizes--tiny)"}}} /-->
```
Comment (jffng) [inline quadrat/block-template-parts/header.html]: Pushed a commit to solve the other part of the issue.

Before | After
------- | ------
Screen Shot 2021-09-30 at 11 11 04 AM | Screen Shot 2021-09-30 at 11 10 48 AM

Comment (scruffian) [reply quadrat/block-template-parts/header.html]: Can we just remove the padding definition altogether?

File: quadrat/sass/templates/_meta.scss (hunk 1 above)
Comment (scruffian) [reply quadrat/sass/templates/_meta.scss]: I think this makes a good case for making the post meta rules opt in in blockbase, as suggested in https://github.com/Automattic/themes/issues/4704

**Comment by MaggieCabrera** [comment]
I'm bringing this in, we can discuss making the post meta opt-in in https://github.com/Automattic/themes/issues/4704

**Pull request merged**